src/rgw/CMakeLists.txt

```diff
@@ -273,6 +273,8 @@ endif()
 if(WITH_RADOSGW_DAOS)
   find_package(DAOS REQUIRED)
   target_link_libraries(rgw_common PRIVATE daos dfs uuid duns)
+  target_include_directories(rgw_common PRIVATE ${PC_DAOS_INCLUDEDIR} )
+  link_directories( ${PC_DAOS_LIBRARY_DIRS} )
```

Comment:
Do we need both of these or just the first one?

Reply:
Both changes are required.  The first helps to find the files it needs, the second one uses the found files in the build

Reply:
One thing to note:  When building this way, the release build will need to modify the rpath of the shared libs in the resulting binary to point to the expected loading path.  Again, building this way is optional, but it does fix the DAOS build issue when assigning the output location of the binaries.  I'm open to other solutions...